src/components/ComputeModal.js

```diff
@@ -1095,14 +1101,45 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       }),
       div({ style: { lineHeight: 1.5 } }, [
         p([
-          `This cloud environment will be created in the region ${computeConfig.computeRegion.toLowerCase()}. `,
-          `Copying data from your workspace bucket in ${bucketLocation.toLowerCase()} may incur network egress charges.`
+          'This cloud environment will be created in the region ',
+          strong(`${computeConfig.computeRegion.toLowerCase()}.`),
+          ' Copying data from your workspace bucket in ',
+          strong(`${bucketLocation.toLowerCase()}`),
+          ' may incur network egress charges.'
```

Comment:
I wonder if we should also mention that network egress charges are not accounted for in cost estimates that are dynamically displayed, which may misleadingly give the impression otherwise.

Same for line 1452.

![image](https://user-images.githubusercontent.com/5438223/142288558-6482efb9-64b3-4374-a922-ce77a145a617.png)

Reply:
Agreed. Something along these lines…

"This cloud environment will be created in the region us-central1. Copying data from your workspace bucket in asia-southeast2 may incur network egress charges. Cost estimates displayed in the Terra cloud environment modal will not reflect these additional charges."

Reply:
Agreed! Added additional text:

> ...may incur network egress charges. Note that network egress charges are not accounted for in cost estimates.